sql/core/src/test/resources/sql-tests/inputs/subquery/exists-subquery/exists-aggregate.sql

```diff
@@ -118,3 +118,10 @@ WHERE  NOT EXISTS (SELECT 1
                                       FROM   dept 
                                       WHERE  emp.dept_id = dept.dept_id 
                                       GROUP  BY dept.dept_id));
+
+-- Window functions are not supported in EXISTS subqueries yet
```

Comment:
Do we plan to make IN and EXISTS subqueries also use the new decorrelation framework?

Reply:
yes this is definitely planned for q3